docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -448,7 +461,7 @@ function yukonca_glider_preprocess_node__multi_step_page(&$variables) {
   foreach ($section_paragraphs as $paragraph) {
     $variables['tabs']['items'][] = [
       'title' => $paragraph->get('field_title')->getString(),
-      'content' => $paragraph->get('field_section_content')->value,
+      'content' => check_markup($paragraph->get('field_section_content')->value, $paragraph->get('field_section_content')->format),
```

Comment:
What issue are these changes related to?

Reply:
This is for - #797